$NetBSD$

The newest git branch command include brackets at the name which needed to be
removed for a valid version name.

--- getversion.orig	2017-08-19 16:40:56.000000000 +0000
+++ getversion
@@ -34,7 +34,7 @@ current_git_branch() {
     # * (no branch)
     #   devel
 
-    GB="$(git branch | awk '$1 == "*" {print $2}')"
+    GB="$(git branch | awk '$1 == "*" {print $2}' | tr -d \()"
     case $GB in
 	"(no" | "(detached" ) echo;;
 	*     ) echo $GB;;
